README.md

```diff
@@ -95,32 +99,39 @@ For example:
 ## Technical documentation
 
 This is a public facing Ruby on Rails application that retrieves browse content from APIs and presents it.
-There is no underlying persistence layer and all content is retrieved from external sources.
+There is no database, and for the majority of pages it serves, collections reads content from external sources.
+
+### Transition landing page ([ /transition ](https://www.gov.uk/transition))
+All content for the transition landing pages are currently read from yaml files. [Welsh](config/locales/cy/coronavirus_landing_page.yml) and [English](config/locales/en/coronavirus_landing_page.yml) translations are available.
+
+### DIT landing page ([ /eubusiness ](https://www.gov.uk/eubusiness))
+All content for the DIT landing pages are currently read from yaml files. [English](config/locales/en/dit_landing_page.yml), [German](config/locales/de/dit_landing_page.yml), [Spanish](config/locales/es/dit_landing_page.yml), [French](config/locales/fr/dit_landing_page.yml), [Italian](config/locales/it/dit_landing_page.yml), [Dutch](config/locales/nl/dit_landing_page.yml) and [Polish](config/locales/pl/dit_landing_page.yml) translations are available.
```

Comment:
```suggestion
All content for the DIT landing pages are currently read from yaml files e.g. for [English](config/locales/en/dit_landing_page.yml). See the [content item's available translations](https://www.gov.uk/api/content/eubusiness) for the current list.
```

This should be slightly more robust for the future 🤞.

Reply:
👍